example/glue/EnvironmentDelegate.cpp

```diff
@@ -132,7 +132,7 @@ MM_EnvironmentDelegate::releaseExclusiveVMAccess()
 
 /**
  * Give up exclusive access in preparation for transferring it to a collaborating thread
- * (i.e. main-to-master or master-to-main). This may involve nothing more than
+ * (i.e. collaborator-to-main or main-to-collaborator). This may involve nothing more than
```

Comment:
This is the only place that was not a direct replacement. Please advise if other terms will make more sense.